packages/bot-web-ui/src/stores/run-panel-store.js

```diff
@@ -187,8 +187,10 @@ export default class RunPanelStore {
         this.dialog_options = {
             title: localize('Are you sure?'),
             message: localize(
-                'This will clear all data in the summary, transactions, and journal panels. All counters will be reset to zero.'
+                'This will reset all counters to zero in summary panel and clear all transactions in transaction panel.'
```

Comment:
Just checking, has this been reviewed by copywriting team?

Reply:
> Just checking, has this been reviewed by copywriting team?

@carol-binary I got this text from [latest design on Zeplin](https://app.zeplin.io/project/5ccc037799118d67c6dc5ba8/screen/5d720f1a8bbca3599aaaa3b9)

![image](https://user-images.githubusercontent.com/60640951/88759180-73148d80-d17f-11ea-9287-4613f363f971.png)

Reply:
This doesn't sound correct, please double confirm with them.

Reply:
@carol-binary , @aaron-binary 
I've asked Dervin and he told our current text is better so I asked Deveraj and lucas to update the design